### PR TITLE
idocs blog

### PR DESCRIPTION
**Pull request by RPrudden**

### CODE REVIEW

File: _posts/2016-03-15-i-docs.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+These are a few of the ideas that I found particularly interesting.
+
+## Agency
+
+_“Critical choices in a strange environment”_
+
+In Thursday’s keynote, New York studio iNKStories presented their upcoming work 1979 Revolution. It’s an example of what they’ve called a verite game, using video game mechanics to communicate a documentary story - in this case, the 1979 Iranian revolution.
+
+Khonsari discussed the importance of agency; allowing the player to make meaningful choices means they will begin to understand the game’s world, simply by seeing the consequences of their actions. This doesn’t work if the consequences are minor or cosmetic, only if they truly affect your options throughout the rest of the game. Because of this, the game has a strongly branching narrative.
```
Comment (jacobtomlinson) [inline _posts/2016-03-15-i-docs.md]: Who is Khonsari?

File: _posts/2016-03-15-i-docs.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+The studio also added a review screen to the end of every chapter, reminding the player of the decisions they had taken and giving some statistics on how popular their choices had been with other players. 
+
+[Read more about branching narratives](http://mycours.es/gamedesign2012/48-2/branching-narrative-from-borges-to-the-hypertext/)
+
+## Non-linear storytelling
+
+_“Most problems that can be solved with linear causal thinking, have been solved by now” - @phivk_
```
Comment (jacobtomlinson) [inline _posts/2016-03-15-i-docs.md]: Can you link to the twitter account?

File: _posts/2016-03-15-i-docs.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+This kind of narrative is familiar and easy to understand. Of course, in the real world it’s rarely so easy to tease out the chain of cause and effect. Most events come about due to a complex interacting network of partial causes. Imagine a journalist writing about a riot - depending on who you speak to, you will get a different idea of the timing and scale, and whether such a thing even occurred. 
+
+Instead of trying to fit events into a pattern of linear cause and effect, non-linear storytelling aims to recognise and embrace these complexities. 
+
+[Read more about linear storytelling](http://meetingpeople.ralphammer.de/2015/12/01/florian-berlin/)
+
+## Web topology
+
+_“Creative interaction forms new associations”_
```
Comment (jacobtomlinson) [inline _posts/2016-03-15-i-docs.md]: You might need an empty line between these quotes to make them break into two paragraphs.

File: _posts/2016-03-15-i-docs.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+_“Relationality - non-linearity - interconnection”_
+
+One example of a non-linear narrative was discussed by Jessica Scarpati in her talk Islands of Impressions. She outlined a textual analysis of the interactive documentary [17000 Islands][islands], which concerns the Indonesian museum park Taman Mini.
```
Comment (jacobtomlinson) [inline _posts/2016-03-15-i-docs.md]: Add a link to Jessica Scarpati?

File: _posts/2016-03-15-i-docs.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+# Main themes from i-Docs
+
+These are a few of the ideas that I found particularly interesting.
+
+## Agency
+
+_“Critical choices in a strange environment”_
+
+In Thursday’s keynote, New York studio iNKStories presented their upcoming work 1979 Revolution. It’s an example of what they’ve called a verite game, using video game mechanics to communicate a documentary story - in this case, the 1979 Iranian revolution.
```
Comment (jacobtomlinson) [inline _posts/2016-03-15-i-docs.md]: Link to the studio?

File: _posts/2016-03-15-i-docs.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+The design emphasises the different possible relations between the clips, and invites the user to explore these by creating their own narratives. 
+
+[Read more about creating nonlinear narratives with web topology](http://alistapart.com/article/building-nonlinear-narratives-for-the-web)
+
+## Exploration
+
+_“Witness or protagonist?”_
+
+Traditionally, virtual reality experiences have not given the user much freedom to explore. You can look in any direction, but you are still rooted to the spot. To some extent this is being addressed using motion tracking in systems like the Vive, but physical restrictions will make it hard to use this to explore anything but very small environments.
```
Comment (jacobtomlinson) [inline _posts/2016-03-15-i-docs.md]: Link to vive?

File: _posts/2016-03-15-i-docs.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+One of the demos, called Assent, used an interesting approach I hadn’t come across before. Focusing on certain (marked) points for a couple of seconds makes you do a kind of zoom/fly maneuver towards that point. It sounds odd, but feels strangely intuitive.
```
Comment (jacobtomlinson) [inline _posts/2016-03-15-i-docs.md]: Link to Assent?

File: _posts/2016-03-15-i-docs.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+One of the demos, called Assent, used an interesting approach I hadn’t come across before. Focusing on certain (marked) points for a couple of seconds makes you do a kind of zoom/fly maneuver towards that point. It sounds odd, but feels strangely intuitive. 
+
+The trailer for Assent didn’t really focus on this, but here is a [video][lands end] of another VR called Land’s End which seems to use the same technique.
+
+## Ownership of a space
+
+_“Characters and stories - you just don’t care!”_
+_“You only care about the space, it’s your space.”_
+
+Francesca Panetta described her experience researching VR techniques for the project [6x9][6x9]. She had shown a VR documentary to a number of people at an arts festival, and surveyed them afterwards to see what they remembered. The answer was, far less than expected. They hadn’t taken in much of the content, and quoted things that hadn’t been in the documentary. Generally, it didn’t seem like they paid much attention to the narrative. All that seemed to interest them was the space itself and their own presence in it.
```
Comment (jacobtomlinson) [inline _posts/2016-03-15-i-docs.md]: Link to Francesca Panetta?

File: _posts/2016-03-15-i-docs.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+
+Traditionally, virtual reality experiences have not given the user much freedom to explore. You can look in any direction, but you are still rooted to the spot. To some extent this is being addressed using motion tracking in systems like the Vive, but physical restrictions will make it hard to use this to explore anything but very small environments.
+
+One of the demos, called Assent, used an interesting approach I hadn’t come across before. Focusing on certain (marked) points for a couple of seconds makes you do a kind of zoom/fly maneuver towards that point. It sounds odd, but feels strangely intuitive. 
+
+The trailer for Assent didn’t really focus on this, but here is a [video][lands end] of another VR called Land’s End which seems to use the same technique.
+
+## Ownership of a space
+
+_“Characters and stories - you just don’t care!”_
```
Comment (jacobtomlinson) [inline _posts/2016-03-15-i-docs.md]: Again line break may help

**Comment by jacobtomlinson** [comment]
Nice article! Could do with more links to the things you mention and maybe a couple of formatting tweaks. See the comments in the commit.

**Comment by RPrudden** [comment]
Thanks for that, those things should be fixed now.

**Comment by jacobtomlinson** [comment]
:+1:

**Pull request merged**